src/messages.js

```diff
@@ -168,7 +169,6 @@ var warnings = {
   W089: "The body of a for in should be wrapped in an if statement to filter " +
     "unwanted properties from the prototype.",
   W090: "'{a}' is not a statement label.",
-  W091: "'{a}' is out of scope.",
```

Comment:
yes, I took my lead from W092 which is missing (https://github.com/jshint/jshint/commit/9b8b363b04112c6cb38ad200124826d1ff0fc37d) but your right, later removals are null. Have changed to null.